src/components/Section.jsx

```diff
@@ -0,0 +1,34 @@
+import { React } from 'react';
+import { Box, Flex, Text } from '@chakra-ui/react';
+import { EditIcon } from '@chakra-ui/icons';
```

Comment:
🚫 **[eslint]** <[import/no-unresolved](https://github.com/import-js/eslint-plugin-import/blob/v2.25.2/docs/rules/no-unresolved.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unable to resolve path to module '@chakra-ui/icons'.